docs/PacketFence_OpenWrt-Hostapd-15-05_Quick_Install_Guide.asciidoc

```diff
@@ -0,0 +1,167 @@
+OpenWrt CHAOS CALMER 15.05 with hostapd Quick Integration Guide
+=================================================================
+:encoding: UTF-8
+:lang: en
+:doctype: book
+:toc: left
+
+include::includes/global-attributes.asciidoc[]
+
+About this Guide
+----------------
```

Comment:
Something is wrong with your formatting in asciidoc please verify, the whole guide appear 'under quotes'